scootapi/setup/builder.go

```diff
@@ -0,0 +1,69 @@
+package setup
```

Comment:
I'd like to see us go all in with code deduping since that was a large motivator for this PR.